modules/transactionpool/update_test.go

```diff
@@ -41,6 +43,38 @@ func (tpt *tpoolTester) testUpdateTransactionRemoval() {
 	if len(tset) != 0 {
 		tpt.assistant.Tester.Error("tset should not have any transactions")
 	}
+
+	// Check that all of the internal maps are also empty.
+	if len(tpt.transactionPool.transactions) != 0 {
+		tpt.assistant.Tester.Error("a field wasn't properly emptied out")
+	}
+	if len(tpt.transactionPool.siacoinOutputs) != 0 {
+		tpt.assistant.Tester.Error("a field wasn't properly emptied out")
+	}
+	if len(tpt.transactionPool.fileContracts) != 0 {
+		tpt.assistant.Tester.Error("a field wasn't properly emptied out")
+	}
+	if len(tpt.transactionPool.siafundOutputs) != 0 {
+		tpt.assistant.Tester.Error("a field wasn't properly emptied out")
+	}
+	if len(tpt.transactionPool.usedSiacoinOutputs) != 0 {
+		tpt.assistant.Tester.Error("a field wasn't properly emptied out")
+	}
+	if len(tpt.transactionPool.newFileContracts) != 0 {
+		tpt.assistant.Tester.Error("a field wasn't properly emptied out")
+	}
+	if len(tpt.transactionPool.newFileContracts) != 0 {
+		tpt.assistant.Tester.Error("a field wasn't properly emptied out")
+	}
+	if len(tpt.transactionPool.fileContractTerminations) != 0 {
+		tpt.assistant.Tester.Error("a field wasn't properly emptied out")
+	}
+	if len(tpt.transactionPool.storageProofs) != 0 {
+		tpt.assistant.Tester.Error("a field wasn't properly emptied out")
+	}
+	if len(tpt.transactionPool.usedSiafundOutputs) != 0 {
+		tpt.assistant.Tester.Error("a field wasn't properly emptied out")
+	}
```

Comment:
blegh. should probably be one `if` statement if you're going to use a generic error message.